docs/tests/integration/post_v1_message-batches/validation.md

```diff
@@ -193,6 +193,69 @@ This test uses the ‘X-Correlation-Id’ header, when provided in a request it
 | 76491414-d0cf-4655-ae20-a4d1368472f3 | Is tested to ensure that when a correlation identifier is sent, we respond with the same value.               |
 
 
+## Scenario: An API consumer submitting a request with an invalid address postcode receives a 400 ‘Invalid Value’ response
+
+A valid contact detail must be structured in this format: { sms: value, email: value, address: { lines: [], postcode: value } }
```

Comment:
Might be worth updating these to appear in code block format, it doesn't appear too readable when the MD files are displayed
![image](https://github.com/user-attachments/assets/d03db6e0-13bc-4f67-9dd4-0959800fd84b)

Raised https://nhsd-jira.digital.nhs.uk/browse/CCM-6748 to address